examples/Resistor_Tutorial/plugs.py

```diff
@@ -0,0 +1,174 @@
+""" my_plugs.py """
+
+import time
+import random
+from typing import Tuple
```

Comment:
This is a deprecated type hint: https://docs.python.org/3/library/typing.html#typing.Tuple
Please use `tuple` directly instead.